src/Symfony/Bundle/FrameworkBundle/Resources/config/console.php

```diff
@@ -129,6 +130,13 @@
             ])
             ->tag('console.command')
 
+        ->set('console.command.dotenv_debug', DotenvDebugCommand::class)
+            ->args([
+                param('kernel.environment'),
+                param('kernel.project_dir'),
+            ])
+            ->tag('console.command', ['isEnabled' => null])
```

Comment:
We use snake case for tag attribute names

```suggestion
            ->tag('console.command', ['is_enabled' => null])
```